src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -88,26 +106,30 @@ where
                     break 'compaction_trigger;
                 }
             };
-            self.pick_and_assign(compaction_group).await;
+            request_channel.unschedule(compaction_group);
```

Comment:
Shall we unschedule after get_compact_task? Looks like all compact request happening before get_compact_task will be considered altogether.